patches/bazel_gazelle/dont-flatten-srcs.diff

```diff
@@ -20,16 +20,16 @@ index 0b23cca..c532ad2 100644
  	return goBinary
```

Comment:
Good news is that at some point in the future we may remove this patch:

https://github.com/bazelbuild/rules_go/pull/3785